runtime/lua/vim/_init_packages.lua

```diff
@@ -53,6 +53,13 @@ require('vim.shared')
 
 vim._submodules = { inspect = true }
 
+local version = require('vim.version')
+vim.version = setmetatable({ cmp = version.cmp, parse = version.parse }, {
+  __call = function()
+    return vim.fn.api_info().version
+  end,
+})
```

Comment:
Integrate this into the block below. We don't want to `require'vim.version'` when we don't use it. The below metatable takes care of that.

```lua
if key == 'version' then
  local version = require('vim.version')    
  t[key] = setmetatable({ cmp = version.cmp, parse = version.parse }, {
  __call = function()
    return vim.fn.api_info().version
  end
  return t[key]
elseif ...
```